src/Raven.Server/Monitoring/Snmp/SnmpHandler.cs

```diff
@@ -94,6 +95,20 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        [RavenAction("/monitoring/snmp/mib", "GET", AuthorizationStatus.Operator)]
+        public async Task GetMib()
+        {
+            var includeServer = GetBoolValueQueryString("includeServer", required: false) ?? true;
+            var includeCluster = GetBoolValueQueryString("includeCluster", required: false) ?? false;
+            var includeDatabases = GetBoolValueQueryString("includeDatabases", required: false) ?? false;
+
+            const string fileName = "RavenDB-MIB.txt";
```

Comment:
Maybe the file name could also include what is included based on the query string parameters?

Reply:
Those files are later copied to snmp/mib folder and the convention is TECHNOLOGY-MIB.txt. I would avoid doing that, especially because later on usage is e.g. `snmpget -v 2c -c ravendb live-test.ravendb.net RAVENDB-MIB::availableMemoryForProcessing`